frontend/package.json

```diff
@@ -64,14 +67,38 @@
     "not ie <= 8"
   ],
   "jest": {
+    "setupFiles": [
+      "<rootDir>/tests/unit/globalSetup.js"
+    ],
     "moduleFileExtensions": [
       "js",
       "json",
-      "vue"
+      "vue",
+      "jsx"
     ],
     "transform": {
-      ".*\\.(vue)$": "vue-jest"
-    }
+      ".*\\.(vue)$": "vue-jest",
+      "^.+\\.vue$": "vue-jest",
+      ".+\\.(css|styl|less|sass|scss|svg|png|jpg|ttf|woff|woff2)$": "jest-transform-stub",
+      "^.+\\.jsx?$": "babel-jest"
+    },
+    "transformIgnorePatterns": [
+      "/node_modules/"
+    ],
+    "moduleNameMapper": {
+      "^@/(.*)$": "<rootDir>/src/$1"
+    },
+    "snapshotSerializers": [
+      "jest-serializer-vue"
+    ],
+    "testMatch": [
+      "**/tests/unit/**/*.spec.(js|jsx|ts|tsx)|**/__tests__/*.(js|jsx|ts|tsx)"
```

Comment:
Not too sure why we're adding jsx support as that's a React specific extension

Reply:
Added by Vue CLI Jest scaffolding (for some reason...). Removing manually then re-running vue add adds it back, so we'll leave it.